src/DatePicker/CalendarYear.js

```diff
@@ -67,27 +69,31 @@ class CalendarYear extends React.Component {
     container.scrollTop = scrollYOffset;
   }
 
-  handleTouchTap = (event, year) => {
-    if (this.props.onYearTouchTap) this.props.onYearTouchTap(event, year);
+  handleTouchTapYear = (event, year) => {
+    if (this.props.onTouchTapYear) this.props.onTouchTapYear(event, year);
   };
 
   render() {
     const years = this.getYears();
     const backgroundColor = this.context.muiTheme.datePicker.calendarYearBackgroundColor;
     const styles = {
-      position: 'relative',
-      height: 'inherit',
-      lineHeight: '36px',
-      textAlign: 'center',
-      padding: '8px 14px 0 14px',
       backgroundColor: backgroundColor,
+      height: 'inherit',
+      lineHeight: '35px',
       overflowX: 'hidden',
       overflowY: 'scroll',
+      position: 'relative',
+      textAlign: 'center',
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
     };
 
     return (
       <div style={styles}>
-        {years}
+        <div>
```

Comment:
This is to center the years when its a ranged datepicker. I've used an extra div to center the years even if there is just three years in the years container of the ranged datepicker.